vtex-apps/menu
# Fix submenu kept open if mouse leaves very fast

#### What problem is this solving?

Submenu kept open if user moves the mouse out of it very quickly.

#### How to test it?

1) Go to https://kiwi--alssports.myvtex.com/
2) Open a submenu with a mouse hover
3) MOVE THE POINTER VERY FASTLY to the bottom of the screen
4) compare with https://alssports.myvtex.com/
5) The submenu should be kept open in the master workspace

#### Screenshots or example usage:

n/a

#### Describe alternatives you've considered, if any.

n/a

#### Related to / Depends on

n/a

#### How does this PR make you feel? [:link:](http://giphy.com/)



![](put .gif link here - can be found under "advanced" on giphy)
